keyboards/keebio/levinson/keymaps/issmirnov/rgb.c

```diff
@@ -0,0 +1,66 @@
+#include "rgb.h"
+
+// TODO gate this debugging header
+#include <print.h>
+
+// Wired up in layer_state_set_user in keymap.c
+uint32_t layer_state_set_rgb(uint32_t state) {
+  static uint8_t old_layer = 255;
+  uint8_t new_layer = biton32(state);
```

Comment:
We've also added a new function that aliases the proper version of `binon##`: 
```suggestion
  uint8_t new_layer = get_highest_layer(state);
```

Reply:
Done.